Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentConsumer/IncidentConsumer.csproj

```diff
@@ -10,13 +10,17 @@
         <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="5.3.0" />
         <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.4.0" />
         <PackageReference Include="Azure.Identity" Version="1.8.0" />
+        <PackageReference Include="Oocx.ReadX509CertificateFromPem" Version="1.2.3" />
+        <PackageReference Include="Microsoft.Identity.Client" Version="4.49.1" />
     </ItemGroup>
     <ItemGroup>
         <None Remove="Microsoft.Azure.WebJobs.Extensions.Storage" />
         <None Remove="Newtonsoft.Json" />
         <None Remove="Microsoft.IdentityModel.Clients.ActiveDirectory" />
```

Comment:
You probably don't need this obsolete library because you switched to a new one, i.e. Microsoft.Identity.Client

Reply:
fixed